presto-testing/src/main/java/io/prestosql/testing/AbstractTestingPrestoClient.java

```diff
@@ -125,6 +126,11 @@ private static ClientSession toClientSession(Session session, URI server, Durati
     {
         ImmutableMap.Builder<String, String> properties = ImmutableMap.builder();
         properties.putAll(session.getSystemProperties());
+        for (Entry<CatalogName, Map<String, String>> catalogAndConnectorProperties : session.getConnectorProperties().entrySet()) {
```

Comment:
I am having hard time understanding that. What are connectorProperties and what are unprocessedCatalogProperties.

Reply:
@losipiuk https://github.com/prestosql/presto/blob/e04cdde8b3e161e304f83ed4c7505ed1a5d107ae/presto-main/src/test/java/io/prestosql/TestSession.java#L45-L64 hth